src/test/java/com/jwplayer/jwplatform/client/AdvertisingClientTest.java

```diff
@@ -0,0 +1,50 @@
+package com.jwplayer.jwplatform.client;
+
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.json.JSONObject;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.jwplayer.jwplatform.exception.JWPlatformException;
+import com.jwplayer.jwplatform.rest.HttpCalls;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({HttpCalls.class})
+public class AdvertisingClientTest {
+
+	AdvertisingClient advertisingClient = AdvertisingClient.getClient("fakeSecret");
+	@Test
+	public void testAllMethods() throws JWPlatformException {
+		advertisingClient.addHeader("test", "testVal");
+		mockStatic(HttpCalls.class);
+		when(HttpCalls.request(anyString(), anyMap(), anyBoolean(), eq("GET"), anyMap())).thenReturn(new JSONObject());
```

Comment:
You can make these test strong by creating a JSONObject() and passing it in to the return. Then you can get it back in each client call and do an assertEquals() to test that you got back the JSONObject you created.

```
JSONObject expectedValue = new JSONObject();
when(HttpCalls.request(anyString(), anyMap(), anyBoolean(), eq("GET"), anyMap())).thenReturn(expectedValue);
JSONObject actualValue = advertisingClient.listAdvertisingSchedules("siteId", new HashMap<>());
assertEquals(expectedValue, actualValue);
```

You actually did this in MediaClientTest.

Comment applies elsewhere.